app/lib/actions/katello/host/hypervisors_update.rb

```diff
@@ -99,9 +100,9 @@ def hypervisors_field(field, hypervisors = @hypervisors)
           hypervisors.map { |h| h[field] }.uniq
         end
 
-        def duplicate_name(hypervisor)
+        def duplicate_name(hypervisor, consumer)
           organization = @organizations[hypervisor[:organization_label]]
-          sanitized_name = ::Katello::Host::SubscriptionFacet.sanitize_name(hypervisor[:name])
+          sanitized_name = ::Katello::Host::SubscriptionFacet.sanitize_name(consumer[:hypervisorId][:hypervisorId])
```

Comment:
Since we could have UUIDs here it would be good to have a test making sure it works: https://github.com/Katello/katello/blob/1ad6c49ef87b4581cc212001119013e1fe9a1db2/test/models/host/subscription_facet_test.rb#L97

Reply:
i added more of an integration test that validates the failing json properly creates two hosts, is that sufficient?

Reply:
works for me